src/tests/ftest/pool/destroy_tests.py

```diff
@@ -373,7 +372,6 @@ def test_destroy_connected(self):
         self.assertTrue(
             exception_detected, "No exception when deleting a connected pool")
 
-    @skipForTicket("DAOS-2741")
     def test_destroy_withdata(self):
```

Comment:
The destroywithdata variant also reports the correct error when trying to destroy the pool.  DER_BUSY

Opening container 63369698-9EF9-4E54-B8EE-5A92C9A137CA
[stdout] DEBUG 19:47:44.344564 drpc_server.go:63: Closing a session
Writing 1 object(s), with 1 record(s) of 9 bytes(s) each, in container 63369698-9EF9-4E54-B8EE-5A92C9A137CA with object class OC_S1
Attempting to destroy a connected pool with data
Expected exception - destroying connected pool with data: Pool destroy returned non-zero. RC: -1012